src/main/scala/subsystem/BaseSubsystem.scala

```diff
@@ -71,6 +73,14 @@ abstract class BaseSubsystem(implicit p: Parameters) extends BareSubsystem {
       }
     }
   }
+
+  class BaseSubsystemLogicalTreeNode extends LogicalTreeNode {
+    override def getOMComponents(resourceBindingsMap: ResourceBindingsMap, cs: Seq[OMComponent]): Seq[OMComponent] = {
```

Comment:
Should this be the default implementation of `getOMComponents` in `LogicalTreeNode`?

Reply:
I don't think so. There are only a few instances where the children are included.